tests/test_parallel_corpus.py

```diff
@@ -40,6 +42,51 @@ def path_to_ft_model():
     return fasttext_model_path
 
 
+def test_clean_text_for_ft():
+    """
+    Test if the text is cleaned as expected.
+    """
+    # Check if '\n' is removed
+    text = 'This is a sample text with new line character.\n'
+    text = clean_text_for_ft(text)
+    assert '\n' not in text
+
+    # Check if all the characters are lower case
+    text = 'This is a sample text with Upper Case.'
+    text = clean_text_for_ft(text)
+    assert text.islower()
+
+    # Check if punctuations and digits are removed
+    text = 'This is 1 and only 1 text with punctuations and digits!!'
+    text = clean_text_for_ft(text)
+    assert '1' not in text and '!' not in text
+
+
+def test_get_ft_lang_code_prob(path_to_ft_model):
+    """
+    Test if the language code and the corresponding probability of texts are as
+    expected.
+    """
+    # Load the FastText's language identification model
+    fasttext_model = fasttext.load_model(path_to_ft_model)
+
+    # Check if an expected exception is raised for a non-string
+    with pytest.raises(AssertionError):
+        text = []
+        _, _ = get_ft_lang_code_prob(text, fasttext_model)
+
+    # Test if '\n' is removed before the prediction
```

Comment:
Could there be some explanation of how this test is testing that `\n` is removed? What would the result have been if `\n` weren't removed?

Reply:
Good point. I have modified this test to reflect your suggestion.